app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -183,6 +191,10 @@ class BackgroundServices(
 
         SyncedTabsIntegration(context, accountManager).launch()
 
+        syncStoreSupport = SyncStoreSupport(syncStore, lazyOf(accountManager)).also {
```

Comment:
can I get a TLDR with what `SyncStoreSupport` does, since it looks like it's on the AC side?

Reply:
sounds good, thank you!